model/src/main/kotlin/OrtResult.kt

```diff
@@ -340,7 +340,7 @@ data class OrtResult(
     /**
      * Return a copy of this [OrtResult] with the [PackageCuration]s replaced by the given [curations].
      */
-    fun replacePackageCurations(curations: List<PackageCuration>): OrtResult =
+    fun replacePackageCurations(curations: Collection<PackageCuration>): OrtResult =
```

Comment:
Just a hint: the result of this function depends on the ordering of the given curations.